assess/scoring/routes.py

```diff
@@ -49,7 +49,7 @@ def score(
     is_rescore = rescore_form.validate_on_submit()
     if not is_rescore and request.method == "POST":
         if score_form.validate_on_submit():
-            current_app.logger.info("Processing POST to {request_path}.", extra=dict(rqeuest_path=request.path))
+            current_app.logger.info("Processing POST to %(request_path)s.", dict(rqeuest_path=request.path))
```

Comment:
Typo in parameter name, should be
`current_app.logger.info("Processing POST to %(request_path)s.", dict(request_path=request.path))`

Reply:
good spot - thanks 🙏